lms/djangoapps/student_account/views.py

```diff
@@ -170,72 +163,43 @@ def _third_party_auth_context(request):
     """
     context = {
         "currentProvider": None,
-        "providers": []
+        "providers": [],
+        "finishAuthUrl": None,
+        "errorMessage": None,
     }
 
-    course_id = request.GET.get("course_id")
-    email_opt_in = request.GET.get('email_opt_in')
-    redirect_to = request.GET.get("next")
-
-    # Check if the user is trying to enroll in a course
-    # that they don't have access to based on country
-    # access rules.
```

Comment:
@bradenmacdonald I'm taking another look and noticed that the new auth completion page doesn't appear to enforce country access/embargo rules by preventing enrollment and redirecting the user to the "blocked" page. The auth completion page should verify that the authenticated user is allowed to access the requested course before enrolling them.